tests/integration_tests/performance/test_boottime.py

```diff
@@ -38,12 +46,18 @@ def test_boottime_no_network(test_microvm_with_boottime):
         vm.log_data)
     print("Boot time with no network is: " + str(boottime_us) + " us")
 
+    return f"{boottime_us} us", f"< {MAX_BOOT_TIME_US} us"
```

Comment:
This message will not appear if the `_test_microvm_boottime` will fail the assert, I d rather place this message to the assert ensuring the boot time is in the desired limits

Reply:
this return is required by the test report so that the pass criteria and results will appear in the json file.
Indeed if the assert fails the details will not appear in the json and this is a limitation of the current implementation.

I added a similar message to the assert nevertheless